install_scripts/ubuntu-bionic-18.04-cloudimg-console.log

```diff
@@ -0,0 +1,665 @@
+[    0.000000] Linux version 4.15.0-91-generic (buildd@lgw01-amd64-013) (gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04.1)) #92-Ubuntu SMP Fri Feb 28 11:09:48 UTC 2020 (Ubuntu 4.15.0-91.92-generic 4.15.18)

```

Comment:
You shouldn't have log files in your repo

Reply:
this log file already existed in the main directory (/vagrant) before my PR. delete him as well?

Reply:
you can use a .gitignore file so that it doesn't upload any log files.

Reply:
Done